### PR TITLE
feat(react): add PdsIcon helper

### PR DESCRIPTION
**Pull request by ju-Skinner**
# Description

Adds PdsIcon helper to the react library. This is required to pass the `icon` property using React SVG imports.

Fixes #(issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

### CODE REVIEW

File: libs/core/src/components/pds-radio/readme.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 | `checked`                  | `checked`        | Determines whether or not the radio is checked.                                                     | `boolean` | `false`     |
 | `componentId` _(required)_ | `component-id`   | A unique identifier used for the underlying component `id` attribute and the label `for` attribute. | `string`  | `undefined` |
 | `disabled`                 | `disabled`       | Determines whether or not the radio is disabled.                                                    | `boolean` | `false`     |
+| `errorMessage`             | `error-message`  | Displays message text describing an invalid state.                                                  | `string`  | `undefined` |
```
Comment (pixelflips) [inline libs/core/src/components/pds-radio/readme.md]: are these changes related?

File: libs/react/src/components/PdsIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+type InternalProps = PdsIconProps & {
+  forwardedRef?: React.ForwardedRef<HTMLPdsIconElement>;
+};
+
+class PdsIconContainer extends React.PureComponent<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+    if (this.props.name) {
+      console.warn(
+        'In Pine React, you import icons from "@ju-skinner/ideal-octo-sniffle" and set the icon you imported to the "name" property. Setting the "name" property has no effect.'
```
Comment (pixelflips) [inline libs/react/src/components/PdsIcon.tsx]: is that the correct location?

**Review by pixelflips — CHANGES_REQUESTED** [review]
A couple of small comments. If those are intentional, just let me know.

**Review by pixelflips — APPROVED** [review]
LGTM!

**Review by anechol — APPROVED** [review]

**Pull request merged**